ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,942 @@
+include::../shared-doc/attributes.adoc[]
+
+= ejb-txn-remote-call: Demonstrates remote EJB calls and transaction propagation
+:author: Ondra Chaloupka
+:level: Intermediate
+:technologies: EJB, JTA, Clustering
+
+[abstract]
+The `ejb-txn-remote-call` quickstart demonstrates the remote EJB calls over two application servers of {productName}.
+
+:standalone-server-type: ha
+:archiveType: war
+:uses-h2:
+:requires-multiple-servers:
+:jbds-not-supported:
+
+== What is it?
```

Comment:
Is this type of heading standard in the quickstarts?  We usually recommend against using questions as headings or titles, as users search for answers rather than questions.  Users are less likely to follow up on search results posed as questions.

Reply:
I reused this heading from other quickstarts - e.g. see https://github.com/wildfly/quickstart/tree/master/helloworld#what-is-it or others. Could be keep as it is or should be changed to something like "Overview"?

Reply:
If this heading is standard in the quickstarts, I would say to leave it so it is consistent with the others.